wp-content/plugins/wporg-learn/inc/redirects.php

```diff
@@ -49,7 +49,7 @@ function wporg_learn_redirect_old_urls() {
 		return;
 	}
 
-	$redirects = array(
+	$pages = array(
```

Comment:
Not all of the redirects in this group are pages, some are post types. If we're going to split out courses we should do the same for tutorials.

Reply:
I agree; that's why I wanted to check with you first. I'll update the PR today to separate these better.